CONTRIBUTING.md

```diff
@@ -164,3 +164,48 @@ the package.
 Fill out the description template in the GitHub interface.
 When you submit the PR, a Heroku review app will be automatically created; it
 will remain available for 5 days.
+
+
+### Deployment 
+
+*Deployment is done from the `master` branch only.*
+
+#### Step 1: Initialize the app on the playground server
+Log in to your account on
+[dash-playground.plotly.host](dash-playground.plotly.host) and create
+a new application. The name of the application should consist of the
+component name in lowercase, with words separated by dashes (`-`). The
+deployment server will provide a remote repository URL that will
+contain the app. Add this to the list of remotes by running `git
+remote add gallery-test [deployment server git URL]`.
+
+#### Step 2: Edit and commit the `Procfile`
+Edit the `Procfile` in the repository to say `gunicorn
+app_name:server`, where `app_name` is the name of the app you want to
+deploy in the `tests/dashbio_demos/` folder. Then, commit the change
+you made to the `Procfile`.
+
+#### Step 3: Push to the playground server
+Run `git push gallery-test master`. This will deploy the app on the
+playground server. Test that it works by visiting the url that is
+displayed in the console.
+
+#### Step 4: Initialize the app on the dash-bio server and push to it 
+Log into the `admin` account on
+[dash-bio.plotly.host](dash-bio.plotly.host) and follow the same
+instructions as in Step 1, but give this remote a different name
+(e.g., by running `git remote add gallery [deployment server git
+URL]`). Then, run `git push gallery master`.
+
+#### Step 5: Undo the `Procfile` commit 
+Run `git log` to find the ID of the commit prior to the one that you
+just made to change the `Procfile`. Then, reset yoru local branch to
+this commit so that the `index.py` app still deploys and runs
+normally. You can do this by running `git reset --hard [commit ID]`.
+
+#### Step 6: Ensure that your branch is even with `master` 
+Since you've reverted the change, running `git diff` should return
+nothing.
+
+#### Step 7: Deploying updates to the `master` branch
+If the `master` branch has been updated, you can push all the changes to the dash-bio remote. However, make sure that the `Procfile` on the dash-bio remote stays the same. 
```

Comment:
Please break this super extra long line! After 80 characters or so (doesn't need to be as strict as code, but still)...

Reply:
@mkcor Fixed in https://github.com/plotly/dash-bio/pull/147/commits/f8b84ec94034d8a36be16bce578acb5a9fa3bda3!